app/controllers/report_results_controller.rb

```diff
@@ -103,8 +103,14 @@ def create
     if @report.has_pit_options?
       pit_date = @result.options['pit_date'].to_date
       chronic_date = @result.options['chronic_date'].to_date
+      project_ids = @result.options.try(:[], 'project_ids')&.reject(&:blank?)&.map(&:to_i) || []
+      project_group_ids = @result.options.try(:[], 'project_group_ids')&.reject(&:blank?)&.map(&:to_i) || []
+      project_ids += GrdaWarehouse::ProjectGroup.
+        joins(:projects).
```

Comment:
Feels like we need a viewable_by on the joins here `merge(GrdaWarehouse::Hud::Project.viewable_by(current_user)`.